lib/features/nems_core/nems_core.module

```diff
@@ -30,9 +30,12 @@ const NEMS_CORE_BLOCK_ACCESS_PERMISSIONS = array(
  * Implements hook_nexteuropa_token_token_handlers().
  */
 function nems_core_nexteuropa_token_token_handlers() {
-  return array(
-    'handler_name' => '\Drupal\nems_core\FileUrlTokenHandler',
-  );
+  // This is shared functionality with mbp_sync, so we check for conflict.
+  if (!class_exists('Drupal\mbp_sync\FileUrlTokenHandler')) {
```

Comment:
@verbruggenalex  and @jonhy81 both are right, I should rename it to make naming consistent but other then that, without the check, it wont work (because of the way the token system is implemented in nexteuropa_token module).
Error: Warning: class_exists() expects parameter 1 to be string, array given 
becasue the value given to class_exists without the check suggested by @jonhy81 is "Array ( [0] => \Drupal\mbp_sync\FileUrlTokenHandler [1] => \Drupal\nems_core\NemsCoreFileUrlTokenHandler )"